src/components/IdScanner/IdScanner.tsx

```diff
@@ -148,7 +148,7 @@ export const IdScanner: FunctionComponent<IdScanner> = ({
 
   useEffect(() => {
     const askForCameraPermission = async (): Promise<void> => {
-      const { status } = await Permissions.askAsync(Permissions.CAMERA);
+      const { status } = await Camera.requestPermissionsAsync();
```

Comment:
Can we use `granted` instead? It seems more appropriate for determining if we have full permissions for the camera! (see [reference](https://docs.expo.io/versions/v42.0.0/sdk/permissions/#permissionresponsegranted))

```suggestion
      const { granted } = await Camera.requestPermissionsAsync();
```

Reply:
Seems like the [documentation](https://docs.expo.io/versions/latest/sdk/camera/) uses this method, let's go with that!